driver/config/config.go

```diff
@@ -1364,11 +1365,13 @@ func (p *Config) WebAuthnForPasswordless(ctx context.Context) bool {
 }
 
 func (p *Config) WebAuthnConfig(ctx context.Context) *webauthn.Config {
+	id := p.GetProvider(ctx).String(ViperKeyWebAuthnRPID)
+	origin := p.GetProvider(ctx).String(ViperKeyWebAuthnRPOrigin)
+	origins := p.GetProvider(ctx).StringsF(ViperKeyWebAuthnRPOrigins, []string{stringsx.Coalesce(origin, id)})
```

Comment:
I think this is a valid strategy. Alternatively this field can also be polymorph `string | []string` but this is probably a cleaner solution and allows us to remove the origin config at some point.

How would we migrate this in Ory Network?

Reply:
I guess we need a new DB column, read from the old one and then only write to the new one. And we need an UI update, right?

Reply:
Or we keep the old DB column, and have the origins comma-separated.

Reply:
New column 👍